src/components/v2/shared/FundingCycleConfigurationDrawers/NFTDrawer/PaymentThresholdFormItem.tsx

```diff
@@ -0,0 +1,42 @@
+import { t } from '@lingui/macro'
+import { Form, FormInstance } from 'antd'
+import InputAccessoryButton from 'components/shared/InputAccessoryButton'
+import FormattedNumberInput from 'components/shared/inputs/FormattedNumberInput'
+import TooltipLabel from 'components/shared/TooltipLabel'
+
+import { NFTFormFields } from './NFTRewardTierModal'
+
+export default function PaymentThresholdFormItem({
+  form,
+}: {
+  form: FormInstance<NFTFormFields>
+}) {
+  const validatePaymentThresholdAmount = () => {
+    const value = form.getFieldValue('paymentThreshold')
+    if (value === undefined || value <= 0) {
+      return Promise.reject(t`Amount required`)
+    }
+    return Promise.resolve()
+  }
+
+  return (
+    <div style={{ display: 'flex' }}>
+      <Form.Item
+        name={'paymentThreshold'}
+        label={
+          <TooltipLabel
```

Comment:
**suggestion** we're using `extra` here: i suggest we remove the tooltip

Reply:
Done